### PR TITLE
This Y Does not exists

### PR DESCRIPTION
**Pull request by Doges**
**Auto-generated PR using AI**

### CODE REVIEW

File: src/commands/misc/tcdne.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
+const { Command, SwitchbladeEmbed } = require('../../')
+
+module.exports = class TCDNE extends Command {
+  constructor (client) {
+    super({
+      name: 'tcdne',
+      aliases: ['thiscatdoesnotexists'],
```
Comment (pedrofracassi) [inline src/commands/misc/tcdne.js:7]:
```suggestion
      aliases: ['thiscatdoesnotexist'],
```

File: src/commands/misc/tpdne.js [hunk 2]
```diff
@@ -0,0 +1,19 @@
+const { Command, SwitchbladeEmbed } = require('../../')
+
+module.exports = class TPDNE extends Command {
+  constructor (client) {
+    super({
+      name: 'tpdne',
+      aliases: ['thispersondoesnotexists'],
```
Comment (pedrofracassi) [inline src/commands/misc/tpdne.js:7]:
```suggestion
      aliases: ['thispersondoesnotexist'],
```

**Comment by metehus** [comment]
> this approve does not exist

so have i approved or not? idk because this comment does not exist

File: src/commands/misc/tpdne.js (hunk 2 above)
Comment (Doges) [reply src/commands/misc/tpdne.js:7]: fu

**Comment by Doges** [comment]
now thats approved

**Comment by Doges** [comment]
> 
> 
> get your english right
> 
> image

**Pull request merged**